sdk/Lusid.Sdk.Tests/tutorials/Ibor/PortfolioCashFlows.cs

```diff
@@ -29,6 +29,48 @@ public void SetUp()
                 _apiFactory.Api<IQuotesApi>(),
                 _apiFactory.Api<IStructuredMarketDataApi>());
         }
+        
+        [Test]
+        public void ExamplePortfolioCashFlowsForBonds()
+        {
+            // CREATE portfolio
+            var portfolioScope = Guid.NewGuid().ToString();
+            var portfolioId = _testDataUtilities.CreateTransactionPortfolio(portfolioScope);
+
+            // CREATE bond
+            var bond = InstrumentExamples.CreateExampleBond() as Bond;
+        
+            // UPSERT bond to portfolio and populating stores with required market data
+            var effectiveAt = new DateTimeOffset(2020, 2, 23, 0, 0, 0, TimeSpan.Zero);
+            _testDataUtilities.AddInstrumentsTransactionPortfolioAndPopulateRequiredMarketData(
+                portfolioScope, 
+                portfolioId,
+                effectiveAt,
+                effectiveAt,
+                bond);
+            
+            // CALL api to get cashflows at maturity
+            var maturity = bond.MaturityDate.Value;
+            var cashFlowsAtMaturity = _transactionPortfoliosApi.GetPortfolioCashFlows(
+                portfolioScope,
+                portfolioId,
+                maturity.AddMilliseconds(-1),
+                maturity.AddMilliseconds(1));
+            
+            // CHECK correct number of cashflow at maturity
+            var expectedNumber = 2;
```

Comment:
I presume this is coupon and redemption notional? if so could we add a comment for the reader

Reply:
Yes, it is the last coupon and the principal returned. I will add a comment and update the PR. 

Thank you for spotting it.